packages/patternfly-4/react-core/src/components/Dropdown/index.js

```diff
@@ -1,5 +1,5 @@
-export { default as Dropdown } from './Dropdown';
-export { DropdownDirection, DropdownPosition } from './dropdownConstants';
+export { default as Dropdown, DropdownWithContext as DefaultDropdown } from './Dropdown';
```

Comment:
Can you remove these exports from index.js.  We don't want these exposed to consumers.